com.unity.netcode.gameobjects/Components/NetworkAnimator.cs

```diff
@@ -485,10 +627,38 @@ private void SendAllParamsAndState()
             }
         }
 
+        private ulong[] GetTargetClientIds(ulong originClientId)
```

Comment:
I'm not sure whether I fully grasp why we are doing this here and need this sort of caching? Don't we just send the update to every client?

Reply:
I feel like there has to be a way to do this in a more simple way. I mean sending RPCs. I saw a similar function for the server above as well. It's a lot of boilerplate just to send an RPC?

Reply:
We're sending messages to every client except:
 - the origin client
 - and the server client (because we're sending this RPC from the server)

Prior version of this code relied on LINQ all the time

Reply:
I am not 100% sure if we need to filter things out like that, but I would think that it conserves bandwidth to not send data to the peers who already have it

Reply:
Okay I think for now this is fine. I hope we can get back at a later point in time and figure out whether there is a way to provide a simple pattern for something like a `EveryoneExceptSelf` RPC.

Reply:
@pdeschain your comment "We're sending"...must include that in the source please

Reply:
And again, since NetworkAnimator should have just the server NA elements, this shouldn't be necessary here, only in CNA

Reply:
Lastly, more like NetworkTransform, I'd prefer were your CNA just ignores what comes its way.  After all, it should have its own local state, and it knows that it's its owner, so it can just skip that and avoid all this bookkeeping, no?  Maybe I'm missing something.